server/paymentProviders/opencollective/virtualcard.js

```diff
@@ -147,14 +149,24 @@ async function create(args, remoteUser) {
     : moment()
         .add(3, 'months')
         .format();
+  // If monthlyLimitPerMember is defined, we ignore the amount field and
+  // consider monthlyLimitPerMember times the months from now until the expiry date
+  let monthlyLimitPerMember;
+  let amount = args.amount;
+  if (args.monthlyLimitPerMember) {
+    monthlyLimitPerMember = args.monthlyLimitPerMember;
+    const monthsFromNowToExpiryDate = Math.round(moment(expiryDate).diff(moment(), 'months', true));
+    amount = Math.round(monthlyLimitPerMember * monthsFromNowToExpiryDate);
```

Comment:
if there is no `expiryDate` it is set by default to 3 months from now(lines 147 to 151) so it would always have the initalBalance setup. Do you think it worths removing it as it seems to work fine? I think this way it makes the code with less conditions to check whether the card will have initialBalance or only monthlyLimitMember. Lmk...